recipes/segyio/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "segyio" %}
+{% set version = "1.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/equinor/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a563f630c959236164b5cf3317671cb0202bdb7cae4f72e71b31a7d9c40365cb
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - setuptools
+    - setuptools_scm
+    - pytest
+    - pytest-runner
+    - numpy
+  run:
+    - python
+    - numpy
```

Comment:
Is this package building against numpy's C API?

If so, this line should be `{{ pin_compatible('numpy') }}`. See [here](https://conda-forge.org/docs/meta.html#building-against-numpy) for details.

Reply:
No, all numpy use is from Python only.